src/sentry/static/sentry/app/components/dataExport.tsx

```diff
@@ -90,12 +92,31 @@ class DataExport extends React.Component<Props, State> {
             {...this.props}
           >
             {children ? children : t('Export All to CSV')}
-          </Button>
+          </StyledBetaButton>
         )}
       </Feature>
     );
   }
 }
 
+const BetaButton = ({children, ...buttonProps}) => (
+  <Button {...buttonProps}>
+    {children}
+    <span id="beta-tag">
+      <BetaTag title="" />
+    </span>
```

Comment:
I'm not the sure this is the most 'proper' way to position the beta tag with CSS but let me know if I should change it.